src/App.js

```diff
@@ -1,13 +1,80 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './Components/GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: null,
+    goodsConfiguration: '',
```

Comment:
do you really need it?

Reply:
it's optional, I use this property for styling buttons to display which list is selected now